examples/misc/demo_ribbon_box.py

```diff
@@ -57,45 +59,35 @@ def draw(self, renderer, *args, **kwargs):
         super().draw(renderer, *args, **kwargs)
 
 
-if True:
-    from matplotlib.transforms import Bbox, TransformedBbox
-    from matplotlib.ticker import ScalarFormatter
-
-    # Fixing random state for reproducibility
-    np.random.seed(19680801)
-
+def main():
     fig, ax = plt.subplots()
 
     years = np.arange(2004, 2009)
-    box_colors = [(0.8, 0.2, 0.2),
-                  (0.2, 0.8, 0.2),
-                  (0.2, 0.2, 0.8),
-                  (0.7, 0.5, 0.8),
-                  (0.3, 0.8, 0.7),
-                  ]
-    heights = np.random.random(years.shape) * 7000 + 3000
-
-    fmt = ScalarFormatter(useOffset=False)
-    ax.xaxis.set_major_formatter(fmt)
+    heights = [7900, 8100, 7900, 6900, 9800]
```

Comment:
```suggestion
    heights = [7900, 8100, 7900, 6900, 9200]
```

With 9800, the label slips out of the axes:
https://6974-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/misc/demo_ribbon_box.html

Reply:
done